### ISSUE TITLE
Cannot build on FreeBSD

### ISSUE THREAD

**Issue by eschnett** [HDFGroup/hdf5#3925]
**Describe the bug**
I am trying to build HDF5 1.14.3 on FreeBSD. This fails with the error message
```
[21:09:25] ../configure: .: line 71: can't open '../config/classic-fflags': No such file or directory
```

This file is indeed not present.

I am configuring with
```sh
[21:09:25]  ---> ../configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --enable-cxx=yes --enable-direct-vfd="$ENABLE_DIRECT_VFD" --enable-fortran=yes --enable-hl=yes --enable-mirror-vfd="$ENABLE_MIRROR_VFD" --enable-parallel="$ENABLE_PARALLEL" --enable-ros3-vfd=yes --enable-static=no --enable-tests=no --enable-tools=yes --enable-unsupported=yes --with-examplesdir=/tmp --with-szlib=${prefix} hdf5_cv_ldouble_to_long_special=no hdf5_cv_long_to_ldouble_special=no hdf5_cv_ldouble_to_llong_accurate=yes hdf5_cv_llong_to_ldouble_correct=yes hdf5_cv_disable_some_ldouble_conv=no hdf5_cv_szlib_can_encode=yes "$(../saved/get_config_setting PAC_C_MAX_REAL_PRECISION ../saved/config.status)" "$(../saved/get_config_setting PAC_FC_ALL_REAL_KINDS ../saved/config.status)" "$(../saved/get_config_setting PAC_FC_MAX_REAL_PRECISION ../saved/config.status)" "$(../saved/get_config_setting PAC_FORTRAN_NUM_INTEGER_KINDS ../saved/config.status)" "$(../saved/get_config_setting PAC_FC_ALL_INTEGER_KINDS ../saved/config.status)" "$(../saved/get_config_setting PAC_FC_ALL_REAL_KINDS_SIZEOF ../saved/config.status)" "$(../saved/get_config_setting PAC_FC_ALL_INTEGER_KINDS_SIZEOF ../saved/config.status)" "$(../saved/get_config_setting PAC_FORTRAN_NATIVE_INTEGER_KIND ../saved/config.status)" "$(../saved/get_config_setting PAC_FORTRAN_NATIVE_INTEGER_SIZEOF ../saved/config.status)" "$(../saved/get_config_setting PAC_FORTRAN_NATIVE_REAL_KIND ../saved/config.status)" "$(../saved/get_config_setting PAC_FORTRAN_NATIVE_REAL_SIZEOF ../saved/config.status)" "$(../saved/get_config_setting PAC_FORTRAN_NATIVE_DOUBLE_KIND ../saved/config.status)" "$(../saved/get_config_setting PAC_FORTRAN_NATIVE_DOUBLE_SIZEOF ../saved/config.status)" "$(../saved/get_config_setting HAVE_Fortran_INTEGER_SIZEOF ../saved/config.status)" "$(../saved/get_config_setting FORTRAN_HAVE_C_LONG_DOUBLE ../saved/config.status)" "$(../saved/get_config_setting FORTRAN_C_LONG_DOUBLE_IS_UNIQUE ../saved/config.status)" "$(../saved/get_config_setting H5CONFIG_F_NUM_RKIND ../saved/config.status)" "$(../saved/get_config_setting H5CONFIG_F_RKIND ../saved/config.status)" "$(../saved/get_config_setting H5CONFIG_F_RKIND_SIZEOF ../saved/config.status)" "$(../saved/get_config_setting H5CONFIG_F_NUM_IKIND ../saved/config.status)" "$(../saved/get_config_setting H5CONFIG_F_IKIND ../saved/config.status)"
```

**Platform (please complete the following information)**
 - HDF5 version 1.14.3
 - FreeBSD 13.2
 - Clang 16.0.6, Gfortran 8
 - Build system (e.g. CMake, Autotools) and version: configure
 - Any configure options you specified
 - MPI library and version (parallel HDF5): MPICH 4.1.3

**Additional context**

**Comment by byrnHDF** [comment]
Yes that is a mistake - it should be intel-fflags in config/freebsd file. It is corrected in develop and soon in hdf5_1_14 branch.

**Comment by lrknox** [comment]
> Yes that is a mistake - it should be intel-fflags in config/freebsd file. It is corrected in develop and soon in hdf5_1_14 branch.

Fix was added to develop in #3883 and has been merged to the hdf5_1_14 branch in #3923 for release inHDF5 1.14.4.
